be/src/exec/pipeline/fragment_executor.cpp

```diff
@@ -135,6 +135,12 @@ Status FragmentExecutor::_prepare_query_ctx(ExecEnv* exec_env, const UnifiedExec
     if (query_options.__isset.enable_profile && query_options.enable_profile) {
         _query_ctx->set_enable_profile();
     }
+    if (query_options.__isset.enable_big_query_profile && query_options.enable_big_query_profile) {
+        _query_ctx->set_enable_big_query_profile();
+    }
+    if (query_options.__isset.big_query_profile_second_threshold) {
```

Comment:
could we only use one flag?  only use big_query_profile_second_threshold? the default valie is 0, means disable. if value larger than zero, means enable big query profile.

Reply:
Yeah, make sense. let me update